rust/headless-client/src/lib.rs

```diff
@@ -135,14 +143,16 @@ struct CliIpcService {
 
 #[derive(clap::Subcommand, Debug, PartialEq, Eq)]
 enum CmdIpc {
-    #[command(hide = true)]
-    DebugIpcService,
-    IpcService,
+    RunDebug,
+    /// Needed to test the IPC service on aarch64 Windows,
+    /// where the Tauri MSI bundler doesn't work yet
+    Install,
+    Run,
```

Comment:
`clippy` started to complain that these all ended in `IpcService` so I changed them.
This means that the CLI args for the IPC service broke compatibility, but I'm pretty sure those args are always in sync because the service is configured during the same MSI / deb install process that updates the exe.